net/ddns-scripts/files/usr/lib/ddns/dynamic_dns_functions.sh

```diff
@@ -589,7 +590,13 @@ verify_host_port() {
 	[ $force_ipversion -ne 0 -a $use_ipv6 -ne 0 -o -z "$__IPV4" ] && __IP=$__IPV6 || __IP=$__IPV4
 
 	if [ -n "$__NCEXT" ]; then	# BusyBox nc compiled with extensions (timeout support)
-		__RUNPROG="$__NC -w 1 $__IP $__PORT </dev/null >$DATFILE 2>$ERRFILE"
+		# nc with zero mode
+		$__NC --help 2>&1 | grep "\-z" >/dev/null 2>&1 && __NCZERO="TRUE"
```

Comment:
You already checked all the --help stuff above -> Here you could just do if [ -n $__NCZERO ]

Reply:
I see that line 582 does the same thing for `__NCEXT`